contracts/governance/FeeSharingProxy.sol

```diff
@@ -49,6 +50,8 @@ contract FeeSharingProxy is SafeMath96, IFeeSharingProxy {
 	uint256 constant FEE_WITHDRAWAL_INTERVAL = 86400;
 
 	uint32 constant MAX_CHECKPOINTS = 100;
+	
+	address public wRBTCAddress;
```

Comment:
we don't need to store it here because `IWrbtcERC20 public wrbtcToken;` is already in the protocol (State.sol contract)
it means no initialization additional param is needed in tests and deployment scripts

Reply:
wrbtcAddress variable has been changed, used the wrbtcToken from protocol instead